### PR TITLE
Changing examples for ENIs and prefixes allocation 

### PR DESCRIPTION
**Pull request by tsahiduek**
This change is proposed to reflect how prefixes and ENIs will be created/allocated as more pods will be scheduled to an instance

**What type of PR is this?**
documentation

**Which issue does this PR fix**:
N/A

**What does this PR do / Why do we need it**:
The existing examples in the table reflect "theoretical" IP assignment, and doesn't reflect how prefixes and ENIs will be allocated as more pods will be scheduled to an instance. 

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:
N/A

**Testing done on this change**:
N/A

**Automation added to e2e**:
N/A

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
N/A

**Does this change require updates to the CNI daemonset config files to work?**:
No

**Does this PR introduce any user-facing change?**:
No. Documentation update

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jayanthvn** [comment]
@tsahiduek - I am on vacation this week and will review it next week. Can you please update your branch?

**Comment by tsahiduek** [comment]
> @tsahiduek - I am on vacation this week and will review it next week. Can you please update your branch?

Done

File: docs/prefix-and-ip-target.md [hunk 1]
```diff
@@ -6,29 +6,39 @@ IPAMD will start allocating (/28) prefixes to the ENIs with `ENABLE_PREFIX_DELEG
...
-| p3dn.24xlarge |         -           |         5       |         10          |  45  |  2   |   45,0       |        4          |      1          |    4,0           |    19     |
-|               |                     |                 |                     |      |      |              |                   |                 |                  |           |
+This table demonstrates how prefixes and ENIs will be allocated and use as pods will be created and scheduled to an instance. When reading this table, please keep in mind the following:
+
+* Every instance type has different limits of ENI pre instance type, and secondary IPv4 addresses per ENI. This information is available on our [EC2 Docs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-eni.html#AvailableIpPerENI).  
+In the example below, `t3.small` was chosen to demonstrate how additional ENIs will be created, since `t3.small` have a maximum of 3 ENIs and 4 IPv4 addresses per ENI. This result in a maximum of 11 available prefixes (3 * 4 minus the primary IPv4 address used bby the instance)
```
Comment (jayanthvn) [inline docs/prefix-and-ip-target.md:14]: bby -> by

**Review by jayanthvn — CHANGES_REQUESTED** [review]
Sorry for the delay. Just one minor change. Rest looks good.

File: docs/prefix-and-ip-target.md (hunk 1 above)
Comment (tsahiduek) [reply docs/prefix-and-ip-target.md]: Thanks. Fixed

Comment (tsahiduek) [reply docs/prefix-and-ip-target.md]: Hi @jayanthvn ,
The typo was fixed. Can you please review it?

**Comment by jayanthvn** [comment]
@tsahiduek - I have updated the branch and changes lgtm.

**Pull request merged**